File: 프로그래머스/lv0/120924. 다음에 올 숫자/다음에 올 숫자.js
function solution(common){
    let answer = 0
    if ((common[0]+common[2])/2 === common[1]){
        console.log("등차수열")
        let d =common[1]-common[0]
        answer = common[common.length -1]+d
    }
    else{
           console.log("등비수열")
        let r = common[1]/common[0]
        answer = common[common.length - 1]*r
    }
     
    return answer
}

File: 프로그래머스/lv0/120924. 다음에 올 숫자/다음에 올 숫자.py
def solution(common):
    answer = 0
    
    if (common[1] - common[0]) == (common[2] - common[1]): # 등차수열
        r = common[1] - common[0]
        return common[len(common) -1] + r
    else:
        r = common[1] / common[0]
        return common[len(common) -1] * r